### ISSUE TITLE
 Can't set timezone-naive datetimes

### ISSUE THREAD

**Issue by Dnomin** [tortoise/tortoise-orm#631]
**Describe the bug**
Can't set timezone-naive datetimes

**To Reproduce**
1. Model with two `DatetimeField`s and type `sqlalchemy.DateTime(timezone=False)`
2. Save timezone-naive datetime to first field (All ok)
3. Save timezone-naive datetime to second field (Error, can't compare aware and naive dts)

`values` from `backends/base/executor.py:execute_update`:
* from first execute: `[datetime.datetime(2021, 2, 2, 0, 0), None]`
* from second: `[datetime.datetime(2021, 2, 2, 0, 0, tzinfo=<UTC>), datetime.datetime(2021, 3, 17, 14, 38, 11)]`

So, as we can see, on second execute, first field value have UTC timezone

**Expected behavior**
Timezone-naive fields

**Additional context**
PostgreSQL : latest
Tortoise-orm : latest
Alembic with SQLAlchemy (for migrations) : latest

**Comment by lsabi** [comment]
Can you be more explicit? It seems strange, but maybe because I'm not understanding the context.

In particular, do you mind posting the model's structure? And explain what you mean with `second execute`? Is it an update query of the same record or a new record?

Thanks

**Comment by xSandie** [comment]
We meet the same strange problem. So I have to roll back to 0.15.18，there is no problem with this version.

**Comment by xSandie** [comment]
I have an old project using the version 0.15.18,  when I upgrade to the latest version 0.16.18, all code with `tortoise.fields.DatetimeField(auto_now_add=True)` have the error `tortoise.exceptions.OperationalError: invalid input for query argument $1: datetime.datetime(2020, 6, 6, 7, 12, 30,... (can't subtract offset-naive and offset-aware datetimes)`

### Possible causes
When I read your code, I found that the class `tortoise.fields.DatetimeField` now using `SQL_TYPE = "TIMESTAMPTZ"` in postgresql [source code,](https://github.com/tortoise/tortoise-orm/blob/dee4de436e8f83e894833bd366d30e0cec400d0c/tortoise/fields/data.py#L29) it will cause some problem with the old project. I will appreciate that if your team could mention this in the changelog.

**Comment by lsabi** [comment]
You have to change the configuration during initialization of tortoise.

Here you see a sample config, though it's without the timezone related configuration, but we'll get there in a moment.

[https://tortoise-orm.readthedocs.io/en/latest/databases.html#passing-in-custom-ssl-certificates](https://tortoise-orm.readthedocs.io/en/latest/databases.html#passing-in-custom-ssl-certificates)

So you have to tune the configuration object mentioned above to get the `datetime` to work properly. There are two settings to be tuned: `use_tz` and `timezone`. You can read more here [https://tortoise-orm.readthedocs.io/en/latest/timezone.html?highlight=timezone#introduction](https://tortoise-orm.readthedocs.io/en/latest/timezone.html?highlight=timezone#introduction).

Here's a sample configuration that you could use, but I didn't try as I only used timezones:

```
{
        "connections": {
            "default": create_dsn(settings)
        },
        "apps": {
            "models": {
                "models": models_list,
                "default_connection": "default"
            }
        },
        "use_tz": False,
        "timezone": "UTC"
    }
```

**Comment by xSandie** [comment]
>     "use_tz": False,
>         "timezone": "UTC"



I tried. It didn't work with postgresql, It seems that I need to change the code in my project 😢

**Comment by lsabi** [comment]
Have you dumped the schema/database before running it?

Also, I'm not sure about the config parameters you have to use, as I only used the `"use_tz": True` since the 0.16.18 release. Try setting it to `True` as well or not passing `timezone` in the configuration, maybe it helps...

**Comment by rafalstapinski** [comment]
Having the same issue,

tl;dr is that the column schema in postgres is `timezone without time zone` but the auto create and auto update columns are _always_ datetime aware

The docs in tortoise orm state 
```
When set use_tz = True, tortoise will always store UTC time in database no matter what timezone set
```
and tortoise achieves this with setting its "now" on auto create and auto update timestamps to be

```
def now() -> datetime:
    """
    Return an aware datetime.datetime, depending on use_tz and timezone.
    """
    if get_use_tz():
        return datetime.now(tz=pytz.utc)
    else:
        return datetime.now(get_default_timezone())
```

As you can see, both of these create timezone-aware datetimes. Because these are timezone aware (despite being UTC), they can not be compared to the database timezone-naive (`timestamp without time zone` in psql) at the adapter (`asyncpg`) level. 

While I also disagree with the general `use_tz` approach taken by tortoise in the first place, the postgres solution here would be to change the `tortoise.timezone.now()` implementation to return `datetime.utcnow()` when `USE_TZ=True`.

Going to see if I can't throw a branch up with that.

**Comment by rafalstapinski** [comment]
https://github.com/rafalstapinski/tortoise-orm/ is the fork i'm currently using with the changes I outlined at the end of my previous comment.

**Comment by weni09** [comment]
when setting use_tz = True,it can transform  value of 'value' to a value with '+0.00' by using method 'timezone.make_aware(value, "UTC")',but this making store of datetime to be failed on mysql(other database have be not testing)!

**Comment by Achilles0509** [comment]
Faced the same issue.
I tried with use_tz=True and use_tz=False options, both are failed.

**Comment by dyuzhou** [comment]
timezone='Asia/Shanghai' ？ 
On PostgreSQL, it seems that no matter how you set it, it always has UTC time

```python
import datetime
from tortoise import timezone

print(datetime.datetime.now())
# > 2023-06-16 15:17:52.114416

print(timezone.now())
# > 2023-06-16 07:17:52.114416+00:00

print(timezone.get_default_timezone())
# > UTC
```

**Comment by cikay** [comment]
looks like it is not going to be solved. So ovveride save method to set update and create fields

```py
from datetime import datetime
from typing import Iterable, Optional

from tortoise.fields import DatetimeField
from tortoise.models import Model
from tortoise.backends.base.client import BaseDBAsyncClient


class BaseModel(Model):
    created_at = DatetimeField(editable=True)
    updated_at = DatetimeField()

    def save(
        self,
        using_db: Optional[BaseDBAsyncClient] = None,
        update_fields: Optional[Iterable[str]] = None,
        force_create: bool = False,
        force_update: bool = False,
    ):
        if not self.id:
            self.created_at = datetime.now()
        self.updated_at = datetime.now()
        return super().save(using_db, update_fields, force_create, force_update)

```

**Comment by CortexPE** [comment]
more dynamic solution:

```py
import typing
from datetime import datetime

from tortoise import Model as BaseModel, fields, BaseDBAsyncClient


class Model(BaseModel):
    async def save(
        self,
        using_db: typing.Optional[BaseDBAsyncClient] = None,
        update_fields: typing.Optional[typing.Iterable[str]] = None,
        force_create: bool = False,
        force_update: bool = False,
    ):
        for column in self._meta.fields_db_projection.keys():
            field_object = self._meta.fields_map[column]
            if not isinstance(field_object, fields.DatetimeField):
                continue
            if field_object.auto_now or (field_object.auto_now_add and not self._saved_in_db):
                setattr(self, column, datetime.now().replace(tzinfo=None))

        return await super().save(using_db, update_fields, force_create, force_update)
```

**Comment by abondar** [comment]
Fixed in 1.0.0
